tools/multipathsecret/multipathsecret/standalone.py

```diff
@@ -0,0 +1,38 @@
+import click
+from .vaultconfig import VaultConfig
+
+
+@click.group()
+@click.option("--vault-address", "-a")
+@click.option("--secret-name", "-n", required=True)
+@click.option("--vault-file", "-v", required=True)
+@click.option("--omit", "-o", multiple=True)
+@click.option("--dry-run", "-x", is_flag=True)
```

Comment:
Documentation strings for these that explain what the parameters are for would be very helpful.

Reply:
Done